src/acl/index.js

```diff
@@ -8,11 +8,18 @@ module.exports = (web3) => {
     return new web3.eth.Contract(require('@aragon/os/build/contracts/ACL').abi, aclAddr)
   }
 
+  const getRoleId = async (repoAddr) => {
+    const repo = new web3.eth.Contract(require('@aragon/os/build/contracts/AragonApp').abi, repoAddr)
```

Comment:
It should be the `Repo` contract rather than `AragonApp`

Reply:
Sure! That's what happens when you don't test this on your local machine :) 

Will fix this.

Reply:
I'm definitely missing what the granting permission in the context of this command and the system means and how all the smart contracts are related to each other, however, maybe you can give me a little pointer on why `repoAddr` is used on an `AragonApp` contract here: https://github.com/aragon/aragon-cli/blob/master/src/acl/index.js#L3 (instead of `Repo` contract)

Reply:
There we just need to use the `kernel()` method which `Repo` inherits from `AragonApp`: https://github.com/aragon/aragonOS/blob/dev/contracts/apm/Repo.sol#L6. That could also be `Repo` instead of `AragonApp` and would still work.